plugin.video.ted-talks/resources/lib/model/language_mapping.py

```diff
@@ -0,0 +1,27 @@
+'''
+Maps between ISO-639-1 language codes and full language names.
+'''
+
+import os
+
+def get_language_code(language):
+    '''
+    This is ludicrous but I can't find XBMC APIs to do it for me :(
```

Comment:
Sadly the API doesn't work very well so I don't want to use it. I've submitted a bug report:

http://trac.kodi.tv/ticket/17231

I should probably check the fix less than 4 years from now :wink: